src/components/DocumentationTopic/Topics.vue

```diff
@@ -28,6 +29,10 @@ export default {
     isSymbolDeprecated: Boolean,
     isSymbolBeta: Boolean,
     sections: TopicsTable.props.sections,
+    topicStyle: {
```

Comment:
Small thing, but you could add a validator that ensures this value is one of the constants you added.